compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -226,6 +226,11 @@ void PreprocessingOptions::bindOptions(OptionsBinder &binder) {
       llvm::cl::desc(
           "File name of a transform dialect spec to use for preprocessing"),
       llvm::cl::cat(category));
+  binder.opt<std::string>(
+      "iree-preprocessing-pdl-spec-filename", preprocessingPDLSpecFilename,
+      llvm::cl::desc(
+          "File name of a transform dialect spec to use for preprocessing"),
```

Comment:
We should document if both TD and PDL preprocessing can be specified at the same time and which one happens first